README.md

```diff
@@ -66,10 +66,12 @@ of NHibernate (in no particular order):
 * [GitHub][] and [SourceForge][] - source code hosting
 * [Atlassian][] - JIRA bug tracker licence and hosting
 * [Log4net][] - logging, by the [Apache Software Foundation][]
-* [JetBrains][] - [ReSharper][] licences for NHibernate developers
+* [JetBrains][] - [ReSharper][] licences for NHibernate developers 
```

Comment:
Strange end of line space add.

Reply:
I wanted to merge TeamCity and ReSharper together on one line, but then decided not to do so now